HAR-3422 Nykytilanteen vilkkuminen

tatut opened the pull request:
**Välitä nykytilanteen karttakuvalle aikavalinta**
Välitetään nykytilanteen karttakuvalle aikavalinnan tuntimäärä, EI alku-
ja loppuaikoja. Koska ajat muuttuvat jokaisella muutoksella, karttakuvaa
pidetään eri tasona ja vanha poistuu aina näkyvistä oli mikä tahansa
muutos kyseessä.

**Käsittele nykytilanteen aikavlainta**

tatut committed:
Välitä nykytilanteen karttakuvalle aikavalinta

Välitetään nykytilanteen karttakuvalle aikavalinnan tuntimäärä, EI alku-
ja loppuaikoja. Koska ajat muuttuvat jokaisella muutoksella, karttakuvaa
pidetään eri tasona ja vanha poistuu aina näkyvistä oli mikä tahansa
muutos kyseessä.

tatut committed:
Käsittele nykytilanteen aikavlainta

tatut committed:
Merge branch 'develop' into HAR-3422

tatut committed:
Merge branch 'develop' into HAR-3422

Kauko commented on src/cljs/harja/tiedot/tilannekuva/tilannekuva.cljs:
```diff
@@ -156,6 +156,19 @@ hakutiheys-historiakuva 1200000)
+   (if (= @valittu-tila :nykytilanne)
+     {:aikavalinta @nykytilanteen-aikasuodattimen-arvo}
+     {:alku (first @historiakuvan-aikavali)
+      :loppu (if (= @valittu-tila :nykytilanne)
```
Tässähän on turha (if)

tatut replied:
totta, en koskenut tuohon else haaraan lainkaan

tatut committed:
Korjaa turhat iffit

tatut committed:
Lisää indikaattori parametri

Indikaattori on vain sen takia, että taso muodostetaan
uudestaan. Toteumissa toteumien määrää käytetään indikaattorina.

tatut committed:
Merge branch 'develop' into HAR-3422

Kauko committed:
Merge branch 'develop' into HAR-3422
